utils/R/convert.input.R

```diff
@@ -6,7 +6,7 @@
 ##' @export
 ##' @author Betsy Cowdery, Michael Dietze
 convert.input <- function(input.id,outfolder,formatname,mimetype,site.id,start_date,end_date,
-                          pkg,fcn,username,con=con,hostname='localhost',write=TRUE,...){
+                          pkg,fcn,username,con=con,hostname='localhost',browndog, write=TRUE,...){
   print(paste("Convert.Inputs",fcn,input.id,hostname))
```

Comment:
As far as I know, the logger functions are actually what I should be using. I'm not sure how much of the printed output we want to keep - I know I need to ultimately go back and remove some of the sanity check printouts. @mdietze @robkooper should I change these to using logger functions?